src/main/java/org/jabref/model/openoffice/uno/DocumentAnnotation.java

```diff
@@ -0,0 +1,15 @@
+package org.jabref.model.openoffice.uno;
+
+import com.sun.star.text.XTextDocument;
+import com.sun.star.text.XTextRange;
+
+/**
+ * Represents a document annotation.
+ *
+ * @param doc    The document
+ * @param name   For the bookmark.
```

Comment:
Please move the comment 

    For the ReferenceMark, Bookmark, TextSection. If the name is already in use, LibreOffice may change the name.

to here.